### ISSUE TITLE
Add a complete message to the compile view

### ISSUE THREAD

**Issue by rorygraves** [scalacenter/scastie#273]
scastie prints:
```
scastie: Connecting.
scastie: Connected.
```
When it is starting a run
it would be great it if printed

```
scastie: Complete
``` 
At the end.

Also possibly - queue - and running.

**Comment by heathermiller** [comment]
Yeah, this would be nice! Indeed I find myself often wondering if Scastie's stalled or something with only "Connecting" and "Connected" messages.

**Comment by MasseGuillaume** [comment]
Totally, I should display something that makes more sense to the user. Connecting/Connected is the state of the Event Stream. It means that we can connect to the web server. From there many things can go wrong and are not reported to the user.

When I implement https://github.com/scalacenter/scastie/issues/270 I will have more information to display.
